src/noisepy/seis/asdfstore.py

```diff
@@ -60,3 +61,85 @@ def _create_station(self, ds: pyasdf.ASDFDataSet, name: str) -> Optional[Station
         except Exception as e:
             logger.warning(f"Missing StationXML for station {name}. {e}")
             return None
+
+
+class ASDFCCStore(CrossCorrelationDataStore):
```

Comment:
This class encapsulates the writing operations for the cross correlations, both to the .h5 and .tmp files.